### PR TITLE
Face mood

### PR DESCRIPTION
**Pull request by hedita**

### CODE REVIEW

File: src/FaceMood.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    name: "Karen",
+    mood: "nice",
+  },
+];
+function FaceMood() {
+  const personList = people.map((person) => {
+    if (person.mood === "happy") {
+      return (
+        <li>
+          {person.name} is {person.mood}. :)
```
Comment (blacksrc) [inline src/FaceMood.jsx]: Please destructure name and mood in function argument.

File: src/FaceMood.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  },
+  {
+    name: "Karen",
+    mood: "nice",
+  },
+];
+function FaceMood() {
+  const personList = people.map((person) => {
+    if (person.mood === "happy") {
+      return (
```
Comment (blacksrc) [inline src/FaceMood.jsx]: Do not repeat the return. Define a variable and name it emoji and fill it based on conditions. Then use that in the return once.

**Review by blacksrc — CHANGES_REQUESTED** [review]

File: src/FaceMood.jsx (hunk 1 above)
Comment (hedita) [reply src/FaceMood.jsx]: Done.

File: src/FaceMood.jsx (hunk 2 above)
Comment (hedita) [reply src/FaceMood.jsx]: Done.

**Pull request merged**